### PR TITLE
Choose option A for style guidelines with popular alternatives.

### PR DESCRIPTION
**Pull request by jjlangholtz**
These two bullets in the style guide had alternatives, I just went ahead and opted for the first choice in both cases but they are completely up for discussion. Thoughts?

### CODE REVIEW

**Comment by vpfaulkner** [comment]
:+1: although you can now remove the sentence about "adopt..." and simply say "Prefer single-quoted strings..." That way it will look the same as every other guideline

File: README.md [hunk 1]
```diff
@@ -3177,19 +3159,6 @@ resource cleanup when possible.
...
 
-  * **(Option B)** Prefer double-quotes unless your string literal
-    contains `"` or escape characters you want to suppress.
-
-    ```Ruby
-    # bad
-    name = 'Bozhidar'
-
-    # good
-    name = "Bozhidar"
```
Comment (markphelps) [inline README.md:3188]: just FYI: http://viget.com/extend/just-use-double-quoted-ruby-strings

It's been shown that double quotes don't actually decrease performance. I would argue that anytime spent thinking or commenting (in code reviews) over double vs single quotes would be a waste and maybe we should just use double quotes by default?

**Comment by jjlangholtz** [comment]
@vpfaulkner I'll make the changes. @markphelps I don't have a strong opinion for either option but I do find myself using double quotes more often than single quotes.

**Comment by markphelps** [comment]
@jjlangholtz same here

**Comment by jjlangholtz** [comment]
Looking through the code the current preference seems to be double quotes in most cases so I will make the adjustment here.

**Pull request merged**